project/ws/app/src/lib/routes/profile-v2/routes/profile-view/profile-view.component.ts

```diff
@@ -12,29 +12,25 @@ import { DateAdapter, MAT_DATE_FORMATS, MAT_DATE_LOCALE } from '@angular/materia
 import _ from 'lodash'
 import moment from 'moment'
 import { Subject } from 'rxjs'
-import { debounceTime, distinctUntilChanged, map, startWith, takeUntil } from 'rxjs/operators'
+import { debounceTime, distinctUntilChanged, startWith, takeUntil } from 'rxjs/operators'
 
-import { ImageCropComponent, ConfigurationsService, ValueService } from '@sunbird-cb/utils';
-import { NsContent, WidgetContentService } from '@sunbird-cb/collection'
+import { ImageCropComponent, ConfigurationsService } from '@sunbird-cb/utils'
 import { LoaderService } from '@ws/author/src/public-api'
 import { PipeCertificateImageURL } from '@sunbird-cb/utils/src/public-api'
 import { IMAGE_MAX_SIZE, PROFILE_IMAGE_SUPPORT_TYPES } from '@ws/author/src/lib/constants/upload'
 import { Notify } from '@ws/author/src/lib/constants/notificationMessage'
 import { NOTIFICATION_TIME } from '@ws/author/src/lib/constants/constant'
-import { HomePageService } from 'src/app/services/home-page.service'
-import { DiscussService } from '../../../discuss/services/discuss.service'
-import { NetworkV2Service } from '../../../network-v2/services/network-v2.service'
 import { UserProfileService } from '../../../user-profile/services/user-profile.service'
 import { OtpService } from '../../../user-profile/services/otp.services'
 
 import { NSProfileDataV2 } from '../../models/profile-v2.model'
-import { NSNetworkDataV2 } from '../../../network-v2/models/network-v2.model'
 import { NsUserProfileDetails } from '../../../user-profile/models/NsUserProfile'
 import { VerifyOtpComponent } from '../../components/verify-otp/verify-otp.component'
 import { TransferRequestComponent } from '../../components/transfer-request/transfer-request.component'
 import { WithdrawRequestComponent } from '../../components/withdraw-request/withdraw-request.component'
 import { NotificationComponent } from '@ws/author/src/lib/modules/shared/components/notification/notification.component'
 import { DesignationRequestComponent } from '../../components/designation-request/designation-request.component'
+import { HomePageService } from 'src/app/services/home-page.service'
```

Comment:
please use relative path

Reply:
I have imported what ever VS code has included

Reply:
it should not be like this